README.md

```diff
@@ -7,11 +7,12 @@ A demo of Python's project with automated tests.
 How to install?
 ===============
 1. Create separate virtual environment with Python 3.5.x.
-2. Run `pip install -r requirements.txt` to install required Python's dependencies.
+2. Run `pip install -r requirements-dev.txt` to install required Python's dependencies.
```

Comment:
We have to support two installations:
- for run - use `requirements.txt`
- for development - use `requirements.txt` and `requirements-dev.txt`